features/product/managing_products/removing_product_price_from_disabled_channel.feature

```diff
@@ -0,0 +1,29 @@
+@managing_products
+Feature: Removing a product's price from disabled channel
+    In order to have only valid prices on enabled channels
+    As an Administrator
+    I want to be able to remove price from disabled channels
+
+    Background:
+        Given the store has currency "USD"
```

Comment:
You didn't specify "GBP" currency in the same way as "USD", however imo this step is not needed for this scenario